py3status/modules/conky.py

```diff
@@ -369,15 +369,18 @@ def post_config_hook(self):
         tmp = f"conky.config = {config}\nconky.text = [[{text}]]"
 
         # write tmp output to '/tmp/py3status-conky_*', make a command
-        self.tmpfile = NamedTemporaryFile(prefix="py3status_conky-", suffix=".conf", delete=False)
+        self.tmpfile = NamedTemporaryFile(
+            prefix="py3status_conky-", suffix=".conf", delete_on_close=False
+        )
         self.tmpfile.write(str.encode(tmp))
         self.tmpfile.close()
         self.conky_command = f"conky -c {self.tmpfile.name}".split()
 
         # skip invalid conky errors
```

Comment:
Please fix this comment too.

Reply:
> Please fix this comment too.

could I just remove the comment?
the new var name seems self-explanatory

Reply:
They seem like debug outputs, here's a test I did:
```
$ conky -c test.conf 
conky: desktop window (0x433) is root window
conky: drawing to desktop window
conky: FOUND: console
conky: FOUND: ncurses
conky: FOUND: file
conky: FOUND: x11
conky: 'i3' x11 session running 'i3' desktop
^Cconky: received SIGHUP, SIGINT, or SIGTERM to terminate. bye!
```

Reply:
Debug/startup outputs aren't supposed to be enabled with console flag on. Seems like the contributors kinda forget or missed that one.

This is really an upstream issue. I am glad we have a solution in place tho.

Anyway, I don't have a preference and this PR is already approved. Thank you for this again. And I see you like this module too (not your first rodeo fixing this). That's great too. 👍😊

Reply:
> Debug/startup outputs aren't supposed to be enabled with console flag on

Yep, that's what I'd think too.
Maybe they have some option to handle log verbosity? but I've no idea if they intended some other behavior for the `"out_to_console": True` option.

> not your first rodeo fixing this

Yeah, it's been part of my i3 config for some time.
It seems I'm at the frontline for bug finding on new updates, I guess I'll see you on the next Fedora release :joy: